SPECS/perl-CPAN-Meta-Check/perl-CPAN-Meta-Check.spec

```diff
@@ -0,0 +1,171 @@
+Name:		perl-CPAN-Meta-Check
+Summary:	Verify requirements in a CPAN::Meta object
+Version:	0.014
+Release:	12%{?dist}
+License:	GPL+ OR Artistic
+Group:          Development/Libraries
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:		https://metacpan.org/release/CPAN-Meta-Check
+Source0:	http://cpan.metacpan.org/authors/id/L/LE/LEONT/CPAN-Meta-Check-%{version}.tar.gz 
+BuildArch:	noarch
+# Build
+BuildRequires:	make
+BuildRequires:	perl
+BuildRequires:	perl(ExtUtils::MakeMaker) >= 6.76
+# Module
+BuildRequires:	perl(base)
+BuildRequires:	perl(CPAN::Meta::Prereqs) >= 2.132830
+BuildRequires:	perl(CPAN::Meta::Requirements) >= 2.121
+BuildRequires:	perl(Exporter)
+BuildRequires:	perl(Module::Metadata) >= 1.000023
+BuildRequires:	perl(strict)
+BuildRequires:	perl(warnings)
+# Test
```

Comment:
If these are test dependencies, can we gate their inclusion at build-time behind the %with_check variable we set during check builds?

Reply:
Added with_check for tests